hazelcast/src/main/java/com/hazelcast/map/impl/recordstore/expiry/ExpirySystem.java

```diff
@@ -334,16 +334,18 @@ public final void evictExpiredEntries(final int percentage, final long now, fina
         tryToSendBackupExpiryOp();
 
         if (logger.isFinestEnabled()) {
-            logProgress(maxScannableCount, scannedCount, expiredCount);
+            logProgress(maxScannableCount, scannedCount, expiredCount, scanLoopStartNanos);
         }
     }
 
-    private void logProgress(int maxScannableCount, int scannedCount, int expiredCount) {
+    private void logProgress(int maxScannableCount, int scannedCount,
+                             int expiredCount, long scanLoopStartNanos) {
         logger.finest(String.format("mapName: %s, partitionId: %d, partitionSize: %d, "
-                        + "remainingKeyCountToExpire: %d, maxScannableKeyCount: %d, "
-                        + "scannedKeyCount: %d, expiredKeyCount: %d"
+                        + "maxScannableCount: %d, scannedCount: %d, expiredCount: %d, "
+                        + "inThePipelineCount: %d, scanTookNanos: %d"
```

Comment:
Is there any reason to change the term from `expiredKeyCount` to `inThePipelineCount`? I don't think we ever used "in the pipeline" for expiry system before and I 'd rather not introduce new terminology that might confuse some users in a patch release.

Reply:
Thanks fixed. BTW It is not `expiredKeyCount` i renamed as `inThePipelineCount`, it is `remainingKeyCountToExpire`, but i agree pipeline is not known terminology.